lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -601,19 +583,9 @@ export class UserAgentApplication {
         reject(ErrorCodes.loginProgressError + Constants.resourceDelimeter + ErrorDescription.loginProgressError);
         return;
       }
```

Comment:
I edited Constants.resourceDelimeter in my PR #579, please check that before merging these changes.

Reply:
This is edited out in a future PR, focus on the changes only please.